switch/dav1d/dav1d.patch

```diff
@@ -15,35 +15,35 @@ index 1104aef..17b804b 100644
  
  
 diff --git a/src/cpu.c b/src/cpu.c
-index d24148c..152fc4b 100644
+index 415266711e..1b8886e837 100644
 --- a/src/cpu.c
 +++ b/src/cpu.c
-@@ -36,6 +36,9 @@
- #elif defined(__APPLE__)
+@@ -37,6 +37,9 @@
+ #ifdef __APPLE__
  #include <sys/sysctl.h>
  #include <sys/types.h>
 +#elif defined(__SWITCH__)
 +#undef PACKED // libdav1d's definition is incompatible with libnx' usage of the macro
```

Comment:
This can be dropped, recent versions of libnx have prefixed the macro with NX_ which solves the issue.